sympy/functions/special/tests/test_delta_functions.py

```diff
@@ -75,7 +77,9 @@ def test_heaviside():
 def test_rewrite():
     x, y = Symbol('x', real=True), Symbol('y')
     assert Heaviside(x).rewrite(Piecewise) == \
-        Piecewise((1, x > 0), (S(1)/2, Eq(x, 0)), (0, True))
+        Piecewise((1, x > 0), (0, x < 0))
+    assert Heaviside(x,y).rewrite(Piecewise) == \
+        Piecewise((1, x > 0), (y, Eq(x, 0)), (0, True))
```

Comment:
Do you think we should special case here for H0==0 and H0==1?   (Pieces don't combine automatically: #8458)